src/modules/tls/tls_rpc.c

```diff
@@ -132,14 +135,29 @@ static void tls_list(rpc_t* rpc, void* c)
 				BUG("failed to convert destination ip");
 			dst_ip[len] = 0;
 			timeout = TICKS_TO_S(con->timeout - get_ticks_raw());
-			rpc->struct_add(handle, "ddsdsd",
+			timestamp = *localtime(&con->timestamp);
+			if (snprintf(timestamp_s, 128, "%d-%02d-%02d %02d:%02d:%02d", timestamp.tm_year + 1900,
+					timestamp.tm_mon + 1, timestamp.tm_mday, timestamp.tm_hour,
+					timestamp.tm_min, timestamp.tm_sec) < 0) {
+				timestamp_s[0] = 'N';
+				timestamp_s[1] = '/';
+				timestamp_s[2] = 'A';
+				timestamp_s[3] = '\0';
+			}
+			rpc->struct_add(handle, "dsdsdsd",
 					"id", con->id,
+					"timestamp", timestamp_s,
 					"timeout", timeout,
```

Comment:
Can we get SNI here as well?

Reply:
you mean to print sni right after the "id"?!

Reply:
Sounds like a good place.

Reply:
:+1: updated PR accordingly